### PR TITLE
MTV-202: "Get logs" button

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MTV-202

Preview: https://deploy-preview-242--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#downloading-logs-ui_forklift

### CODE REVIEW

**Comment by apinnick** [comment]
@istein1 @mguetta1 
Sorry about the short notice. Ready for QE.

File: documentation/modules/accessing-logs-ui.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+You can download logs for a completed, failed, or canceled migration plan and for migrated virtual machines (VMs) by using the {project-short} web console.
+
+.Procedure
+
+. In the web console, click *Migration plans*.
+. To download logs for a migration plan, click *Get logs* beside the migration plan name.
+. To download logs for a migrated VM, click the migration plan name and then click *Get logs* beside the VM.
++
+You can download the VM logs while the migration plan is running.
```
Comment (mguetta1) [inline documentation/modules/accessing-logs-ui.adoc:16]: The `Get logs` button beside the VM is disabled while the plan is running, and it has a tooltip that says:
"cannot run must gather until the migration is finished"

File: documentation/modules/accessing-logs-ui.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+. In the *Get logs* window, click *Get logs*.
++
```
Comment (mguetta1) [inline documentation/modules/accessing-logs-ui.adoc:19]: - Once pressing on Get logs in the Get logs window, the logs are being collected and a message pops up once it is done:
"Log collection complete."
- 'Download logs' button replaces the 'Get logs' button and the user have to press on it to download the logs (archived file)

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/accessing-logs-ui.adoc (hunk 2 above)
Comment (apinnick) [reply documentation/modules/accessing-logs-ui.adoc:19]: I was using Vince's mockups. I will check the forklift-ui repo

File: documentation/modules/accessing-logs-ui.adoc (hunk 1 above)
Comment (apinnick) [reply documentation/modules/accessing-logs-ui.adoc:16]: OK. In the mockups, it says that you can download a VM log while the plan is running. I'll look into this.

**Comment by apinnick** [comment]
@mguetta1 PR has been updated with your comments. Please review.

**Review by mguetta1 — APPROVED** [review]
Looks good to me

**Pull request merged**